common/app/views/fragments/bashers/ThisIsTheNHS.scala.html

```diff
@@ -0,0 +1,41 @@
+@()(implicit request: RequestHeader)
+
+<div class="gs-container">
+    <div class="this-is-the-nhs">
+        <h1 class="title title--first">this is</h1>
+        <h1 class="title title--second">the NHS</h1>
```

Comment:
They're both absolutely positioned relative to the parent though. Wouldn't nesting the second line inside the first, mess this up?